Pop-up remains after mouse cursor leaves window

editorsdiary opened the pull request:
![tw230128-1](https://user-images.githubusercontent.com/43783010/215268027-82961d12-3b62-4870-9371-3d08dc647c42.jpg)

The attached picture shows the mouse cursor moving from left to right, passing over the "fleet" sub-window, and now the cursor is over the equipment column in the main browser.

In this state, it is still possible to control the game by clicking on the icons in the hidden area, but it is very inconvenient for me because I cannot see the screen.

This does not happen all the time, but to get rid of it, I have to move the cursor back to the "fleet" sub-window and turn it off.

Can I avoid this phenomenon by changing the settings?

Or does this not happen in other people's environments?

myangelkamikaze committed:
add max equip name width to fleet image generator

myangelkamikaze committed:
use preview lang version

myangelkamikaze committed:
fix equipment name alignment in fleet image generator

myangelkamikaze committed:
add game asset downloader service

myangelkamikaze committed:
add setting for downloading missing ship images in fleet image generator

myangelkamikaze committed:
remove unnecessary raw string literal

myangelkamikaze committed:
add image export to file to fleet image generator

myangelkamikaze committed:
clean up ship encyclopedia translations

myangelkamikaze committed:
add parameter name to ship name tooltip in fleet view

myangelkamikaze committed:
use TabControl in fleet image generator config

myangelkamikaze committed:
add AACI 27

myangelkamikaze committed:
use the new fleet image generator in fleet view

myangelkamikaze committed:
5.1.11.0

myangelkamikaze committed:
experimental: api logging

myangelkamikaze committed:
5.1.11.1

myangelkamikaze committed:
fix cast exception when trying to unmute with keyboard
fixes #233

myangelkamikaze committed:
fix volume manager init logic

myangelkamikaze committed:
5.1.11.2

myangelkamikaze committed:
add a workaround so browser doesn't crash in case of WMI problems (#231)

myangelkamikaze committed:
don't rethrow...

myangelkamikaze committed:
fix api observer restart

myangelkamikaze committed:
delete upstream proxy when it gets disabled

myangelkamikaze committed:
use bigger font size for fleet image generator font size

myangelkamikaze committed:
use extension methods

myangelkamikaze committed:
fix opening ASW condition

gre4bee committed:
Fix for Replays not using the correct node ID (#236)

myangelkamikaze committed:
fix exp checker texts

myangelkamikaze committed:
add wpf exp checker

myangelkamikaze committed:
prevent opening fleet image generator before ship data is loaded

myangelkamikaze committed:
fix typo

myangelkamikaze committed:
add option to use title as file name to fleet image generator

myangelkamikaze committed:
5.1.11.3

myangelkamikaze committed:
save layout before reloading it for layout locking

myangelkamikaze committed:
fix crash when loading invalid event lock planner

myangelkamikaze committed:
save layout before reloading it when changing grid splitter size

myangelkamikaze committed:
update ship id enum

myangelkamikaze committed:
update equipment id enum

myangelkamikaze committed:
remember max aircraft proficiency setting for fleet and AB

myangelkamikaze committed:
localize ship types in battle view

myangelkamikaze committed:
add compact mode to battle view (#145)

myangelkamikaze committed:
slight adjustment to battle view so compact mode columns can stay aligned

myangelkamikaze committed:
5.1.11.4

myangelkamikaze committed:
rework fleet image generator fleet handling logic to avoid render issues (might be related to #237)

myangelkamikaze committed:
add padding to the comment input in fleet image generator so the text doesn't get hidden behind the scrollbar

myangelkamikaze committed:
add option for how many columns there should be for fleets in fleet image generator

myangelkamikaze committed:
fix image stretch bug in fleet image generator

myangelkamikaze committed:
center air bases in fleet image generator

myangelkamikaze committed:
fix opening ASW condition for oilers

myangelkamikaze committed:
add opening ASW tests for oilers

myangelkamikaze committed:
fix master data update logic

myangelkamikaze committed:
update test data

myangelkamikaze committed:
update ship parameter record

myangelkamikaze committed:
5.1.12.0

myangelkamikaze committed:
update ship parameter record

myangelkamikaze committed:
add new ship classes

Jebzou committed:
Battle detail WPF (#240)

* Battle detail WPF

myangelkamikaze committed:
fix and document short ship types

Jebzou committed:
Update Sonarless OASW condition for Samuel B Roberts

myangelkamikaze committed:
add wpf config

myangelkamikaze committed:
extract file service

myangelkamikaze committed:
add wpf connection config
added a warning when port and upstream proxy port are the same (#150)

myangelkamikaze committed:
fix naming and project structure

Jebzou committed:
Added pumpkin

myangelkamikaze committed:
add font overrides

myangelkamikaze committed:
add wpf ui config

myangelkamikaze committed:
refactor configurations to user controls

myangelkamikaze committed:
add wpf log config

myangelkamikaze committed:
clean up log config

myangelkamikaze committed:
remove play time properties

myangelkamikaze committed:
add wpf behavior config

myangelkamikaze committed:
remove outdated todo

myangelkamikaze committed:
add wpf debug config

myangelkamikaze committed:
add wpf window config

myangelkamikaze committed:
add wpf subwindow config

myangelkamikaze committed:
add display attribute to enums

myangelkamikaze committed:
add missing localization string

myangelkamikaze committed:
update configuration notifier translations

myangelkamikaze committed:
add missing AB notification translation

myangelkamikaze committed:
add wpf notification config (#246)

myangelkamikaze committed:
5.1.12.1

myangelkamikaze committed:
update late model torpedo check

myangelkamikaze committed:
formatting

myangelkamikaze committed:
add wpf bgm config

myangelkamikaze committed:
disable bgm config column persistence

myangelkamikaze committed:
make ConfigRes public

myangelkamikaze committed:
add wpf fleet config

myangelkamikaze committed:
add wpf arsenal config

myangelkamikaze committed:
add wpf dock config

myangelkamikaze committed:
formatting

myangelkamikaze committed:
refactor checkbox enum wrapper

myangelkamikaze committed:
add wpf headquarters config

myangelkamikaze committed:
add wpf compass config

myangelkamikaze committed:
add wpf quest config

myangelkamikaze committed:
add wpf group config

myangelkamikaze committed:
add wpf combat config

myangelkamikaze committed:
formatting

myangelkamikaze committed:
fix folder picker relative path

myangelkamikaze committed:
add wpf browser config

myangelkamikaze committed:
add wpf ab config

myangelkamikaze committed:
add wpf json config

gre4bee committed:
APIObserver Localization (#247)

myangelkamikaze committed:
5.1.12.2

myangelkamikaze committed:
fix font initialization logic

myangelkamikaze committed:
localize font names

myangelkamikaze committed:
improve font detection logic

Jebzou committed:
Equipment picker (#248)

Part of #245

myangelkamikaze committed:
add missing resource

myangelkamikaze committed:
add kancolle api project

myangelkamikaze committed:
rework api file saving

myangelkamikaze committed:
add sorties to database

myangelkamikaze committed:
add database explorer

myangelkamikaze committed:
add missing command

myangelkamikaze committed:
fix translations

myangelkamikaze committed:
5.1.12.3

myangelkamikaze committed:
refactor api file explorer

myangelkamikaze committed:
add sortie explorer

gre4bee committed:
Empty Supply Indication (#251)

--added blinking background to indicate empty fuel/ammo bar

myangelkamikaze committed:
5.1.13.0

myangelkamikaze committed:
remove old tools

myangelkamikaze committed:
5.1.13.1

myangelkamikaze committed:
remove unused dependency

myangelkamikaze committed:
don't rethrow when failing to load layout

myangelkamikaze committed:
update IsAbyssalEquipment

myangelkamikaze committed:
filter out unknown ship type

myangelkamikaze committed:
add missing daihatsu expedition bonus

gre4bee committed:
Add Background Color by world similar to quest view (#256)

Jebzou committed:
Equipment upgrade planner (#250)

part of #245

gre4bee committed:
Add TP for FleetImageGenerator (#257)

myangelkamikaze committed:
add unassigned to equipment upgrade planner

myangelkamikaze committed:
fix equipment assigning logic bug

myangelkamikaze committed:
simplify linq query

myangelkamikaze committed:
fix Japanese text for adding equipment from library

myangelkamikaze committed:
add equipment upgrade plan assignment tooltip

myangelkamikaze committed:
5.1.13.2

myangelkamikaze committed:
fix equipment assigning logic bug (2)

myangelkamikaze committed:
update nuget packages

myangelkamikaze committed:
update mvvm toolkit

myangelkamikaze committed:
update min webview2 version

myangelkamikaze committed:
remove unused enum

myangelkamikaze committed:
add logic for downloading missing files

myangelkamikaze committed:
Kancolle api types (#260)

myangelkamikaze committed:
fix merge bug

Jebzou committed:
Equipment upgrade planner Part 2 (#258)

part of #245

myangelkamikaze committed:
Revert to CefSharp (#263)

gre4bee committed:
Removed limit fps and disable gpu compositing commandlineargs for framerate

gre4bee committed:
Adjust WindowlessFrameRate for high refresh monitors

gre4bee committed:
remove keydown for f7 and f5 as its handled in xaml

gre4bee committed:
Remove previewkeydown event for cefsharp

Jebzou committed:
Equipment upgrade planner Part 2 (bonus) (#265)

Jebzou committed:
Equipment upgrade planner Part 3  (#266)

Jebzou committed:
Small fix to equipment upgrade plan viewer

myangelkamikaze committed:
fix text color

myangelkamikaze committed:
add browser option

myangelkamikaze committed:
add * to indicate EO needs to get restarted after switching the browser

myangelkamikaze committed:
Sortie data rework (#268)

myangelkamikaze committed:
5.1.13.3

Jebzou committed:
If upgrade goal reached, cost should be 0

Jebzou committed:
Equipment upgrade plans dont get refreshed when adding a plan if there's a day filter

gre4bee committed:
CefSharp HwndHost ver (#270)

myangelkamikaze committed:
5.1.13.4

myangelkamikaze committed:
fix screenshots

myangelkamikaze committed:
fix last screenshot

myangelkamikaze committed:
rework cef keyboard handler so that the correct language can be used

myangelkamikaze committed:
Revert "5.1.13.4"

This reverts commit 0b24a2341ae0921a1c2f44109f6c67518e9f736d.

myangelkamikaze committed:
5.1.13.4

myangelkamikaze committed:
fix keyboard handler

myangelkamikaze committed:
fix update check

Jebzou committed:
Add a way to filter upgrades on current day in upgrade planner (#271)

gre4bee committed:
Use old cookie method with dynamic javascript date (#272)

myangelkamikaze committed:
use the winforms cefsharp version (#273)

myangelkamikaze committed:
add expeditions to database

myangelkamikaze committed:
5.1.13.5

myangelkamikaze committed:
update wiki data parser

myangelkamikaze committed:
update generated data

myangelkamikaze committed:
update ship parameter record

myangelkamikaze committed:
update parameter merging logic - assume default parameter record contains the most recent data

myangelkamikaze committed:
resource bars shouldn't blink on escaped ships

myangelkamikaze committed:
storyboard experiment

Jebzou committed:
Current day display of upgrade plans doesn't refresh on day changed (#275)

gre4bee committed:
Fix ExtraBrowser enter button for address bar (#274)

myangelkamikaze committed:
add datetime filter to sortie records

myangelkamikaze committed:
5.2.0.0

myangelkamikaze committed:
fix keyboard handler

myangelkamikaze committed:
adjust equipment list name sort behavior

gre4bee committed:
Opening ASW conditions update (#279)

myangelkamikaze committed:
fix localization

Jebzou committed:
Display a tooltip when hovering upgrade plan viewer (#278)

Jebzou committed:
Equipment count logic for upgrades fixed

myangelkamikaze committed:
fix CefSharp volume

myangelkamikaze committed:
fix floating equipment upgrade plan viewer crashing the app when reloading layout

myangelkamikaze committed:
fix CefSharp cache path

myangelkamikaze committed:
fix CefSharp volume initialization

myangelkamikaze committed:
downgrade appcenter

looks like the newest version breaks on win7

myangelkamikaze committed:
optimize sortie record viewer db access

myangelkamikaze committed:
add null or empty check

myangelkamikaze committed:
add quick config access and foreground/background customizations to fleet image generator

myangelkamikaze committed:
update cefsharp

myangelkamikaze committed:
5.2.0.1

myangelkamikaze committed:
text fix

myangelkamikaze committed:
fixes #269

myangelkamikaze committed:
filter flagships by flagship type

myangelkamikaze committed:
5.2.1.0

myangelkamikaze committed:
senka viewer

myangelkamikaze committed:
add quest senka to senka viewer

myangelkamikaze committed:
handle no api file in senka viewer

myangelkamikaze committed:
update equipment id enum

myangelkamikaze committed:
update deptch charge list

myangelkamikaze committed:
localization fixes

myangelkamikaze committed:
fix mapping

Jebzou committed:
Ship picker (#289)

Jebzou committed:
Create sonarcloud.yml

Jebzou committed:
Training planner (#288)

myangelkamikaze committed:
remove texts

myangelkamikaze committed:
5.2.1.1

Jebzou committed:
Update sonarcloud.yml

Jebzou committed:
Update sonarcloud.yml

Jebzou committed:
Update sonarcloud.yml

Jebzou committed:
Remove dependency on Windows Media Player (#291)

gre4bee committed:
Add propertychanged for RemainingExpToTarget (#292)

Jebzou committed:
Code quality  (#294)

gre4bee committed:
Add predicted battle rank to information view (#295)

myangelkamikaze committed:
fix validation

myangelkamikaze committed:
remove using

myangelkamikaze committed:
add quality gate status badge

Jebzou committed:
Fix media player
- Sound could only played once

Jebzou committed:
Merge branch 'main' of https://github.com/ElectronicObserverEN/ElectronicObserver

myangelkamikaze committed:
properly handle failed data download

Jebzou committed:
Scrapping a ship doesn't remove it's training plan

Jebzou committed:
Merge branch 'main' of https://github.com/ElectronicObserverEN/ElectronicObserver

myangelkamikaze committed:
fix training plan notifications

myangelkamikaze committed:
add 7-5 tasks

myangelkamikaze committed:
fix kanmusu list export

myangelkamikaze committed:
add bulge ship filter

myangelkamikaze committed:
fix quest tracker progress bug

Jebzou committed:
Finished training plans should dissapear if "Display finished plans" is uncheck (#296)

myangelkamikaze commented:
You should open an issue here: https://github.com/ElectronicObserverEN/ElectronicObserver/issues

But no, this isn't the intended behavior and I didn't see it so far. You're using Windows 10 right?

myangelkamikaze committed:
add ASW plane aircraft level bonus

myangelkamikaze committed:
Revert "add ASW plane aircraft level bonus"

This reverts commit 834dbb44c1ec819de9f10051e0c4c8a253588c98.

myangelkamikaze committed:
Fix senka viewer

myangelkamikaze committed:
fix training plan adding

myangelkamikaze committed:
senka viewer logic rework

Jebzou committed:
Add pagination to the Equipment Upgrade Planner (#300)

gre4bee committed:
Adjust Senka Cutoff (#301)

gre4bee committed:
Fix LocalTime for ResourceRecord

myangelkamikaze committed:
senka viewer adjustments

myangelkamikaze committed:
fix current senka

myangelkamikaze committed:
5.2.1.2

myangelkamikaze committed:
null annotations

myangelkamikaze committed:
add missing translations

myangelkamikaze committed:
minor sortie record rework

gre4bee committed:
Fix for senka viewer DateEnd and DateBegin and Sync with HQ Level tooltip for senka

--Fix for senka viewer DateEnd and DateBegin
-- Adjust to synce with HQ Level tooltip

myangelkamikaze commented:
@editorsdiary you should close this PR, since it's not related to this project
